.github/workflows/tag_image_push.yml

```diff
@@ -239,41 +246,14 @@ jobs:
             ${TAGS_ARG} \
             --platform=${PLATFORMS} $DOCKER_DIR
 
-      - name: Slack notification
-        uses: ./.github/actions/slack-notification
-        if: failure()
-        with:
-          slack-webhook-url: ${{ secrets.SLACK_WEBHOOK }}
-
-  post-push:
-    runs-on: ubuntu-latest
-    needs: [ push, prepare ]
-    steps:
-      - uses: actions/checkout@v4
-        with:
-          sparse-checkout: .github
-      - name: Generate Docker Hub Description
-        run: |
-          .github/scripts/generate-docker-hub-description.sh
-      - name: Update Docker Hub Description of OSS image
-        if: needs.prepare.outputs.should_build_oss == 'yes'
-        uses: peter-evans/dockerhub-description@v4
-        with:
-          username: ${{ secrets.DOCKER_USERNAME }}
-          password: ${{ secrets.DOCKER_PASSWORD }}
-          repository: hazelcast/hazelcast
-          short-description: Hazelcast Docker Image
-          readme-filepath: ./README-docker.md
-
-      - name: Update Docker Hub Description of EE image
-        if: needs.prepare.outputs.should_build_ee == 'yes'
-        uses: peter-evans/dockerhub-description@v4
+      - name: Update Docker Hub Description
+        if: needs.prepare.outputs.should_build_readme_ee == 'yes' || needs.prepare.outputs.should_build_readme_oss == 'yes'
```

Comment:
Nit - if your going to check these in the action, why check here as well?
If both are false, we run the action pointlessly but it doesn't do much beyond a sparse checkout.

Reply:
To see easily that this action hasn't been called, but I've changed the action as we need to run it always for both images as they share the same description: https://github.com/hazelcast/hazelcast-docker/pull/793/commits/359118f445ab86e7423147b64dd5e2eb58c2ce0f